capstone-frontend/src/components/BookSearchList.jsx

```diff
@@ -7,26 +7,51 @@ class BookSearchList extends Component {
   constructor(props) {
     super(props);
     this.state = {
-      bookList: []
+      bookSearchList: []
```

Comment:
nit: This will store a list of books that come back from the search right? In that case I think `bookList` or just `books` actually makes more sense here, particularly since `BookSearchList` is already the name of the component.

Reply:
I think I'll use `books`. Mauro and I thought that `bookList` was ambiguous since he will be implementing a list of `bookLists` in this state as well so deciphering between bookList and bookLists (or similar) would get confusing.